### DISCUSSION TITLE
User login API

### DISCUSSION THREAD

**Discussion by matteovisotto** [modoboa/modoboa#2100] [Help]
I need to integrate email users with other applications.
Is there any way to have a public API where I can send login request?
Or using Modoboa with open directory ?

Thanks

**Comment by tonioo** [comment]
There is no login API for the moment. What do you call open directory?

**Reply by matteovisotto** [reply]
Sorry, Active Directory (Samba on Linux)

**Reply by tonioo** [reply]
You can tell Modoboa to use an LDAP directory as authentication provider (https://modoboa.readthedocs.io/en/latest/configuration.html#ldap) and you can also enable synchronization (https://modoboa.readthedocs.io/en/latest/configuration.html#ldap-synchronization)
